x-pack/test/functional_with_es_ssl/apps/cases/view_case.ts

```diff
@@ -282,22 +282,14 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
 
           await header.waitUntilLoadingHasFinished();
 
-          const propertyActions = await find.allByCssSelector(
-            '[data-test-subj*="property-actions-ellipses"]'
-          );
-
-          propertyActions[propertyActions.length - 1].click();
```

Comment:
This was trying to account for the description ellipses, now we can call it out directly.